tika-core/src/main/java/org/apache/tika/language/detect/LanguageNames.java

```diff
@@ -42,13 +42,13 @@ public static String normalizeName(String languageTag) {
     }
 
     public static boolean isMacroLanguage(String languageTag) {
-        Locale locale = Locale.forLanguageTag(languageTag);
+        // Locale locale = Locale.forLanguageTag(languageTag); unused
```

Comment:
Why not just remove?

Reply:
My understanding is that this was kept there because of the TODO below, i.e. this is part of an incomplete piece of code.

Reply:
Correct: there is a todo below